### ISSUE TITLE
what type of GPU you are using?

### ISSUE THREAD

**Issue by JarvisAI-7** [zhengziqiang/CoralSCOP#4]
Hello! May I ask what type of GPU you are using? Approximately how much time would it take with a 4090?

**Comment by zhengziqiang** [comment]
I am using a single Gtx 3090 gpu. The time cost also depends on the image resolution and the point number you are choosing. It usually takes less time to get there on 4090 than on 3090.

**Comment by JarvisAI-7** [comment]
> I am using a single Gtx 3090 gpu. The time cost also depends on the image resolution and the point number you are choosing. It usually takes less time to get there on 4090 than on 3090.

However, I later saw in the paper that six A100 GPUs were used, and I've tried many methods to install pycocotools-2.0.4.tar.gz without success. I'm working on a remote GPU platform; I downloaded the file and attempted to install it using `pip install ./pycocotools-2.0.4.tar.gz --no-cache-dir --timeout 120`.

**Comment by zhengziqiang** [comment]
Sorry, I misunderstood that you were asking the GPU what I am using for inference. I used A100 for training. 

For `pycocotools`, I think you don't need to install the specific version, please just use `pip install pycocotools` to install it.

**Comment by JarvisAI-7** [comment]
> Sorry, I misunderstood that you were asking the GPU what I am using for inference. I used A100 for training.
> 
> For `pycocotools`, I think you don't need to install the specific version, please just use `pip install pycocotools` to install it.

![image](https://github.com/user-attachments/assets/e9f7efdc-71b4-49f9-89c1-61a290fa328e)
"I noticed your YAML file specifies versions, and after a day of trying, I finally succeeded. Is this outcome expected? I have reduced the dataset."

**Comment by JarvisAI-7** [comment]
> > Sorry, I misunderstood that you were asking the GPU what I am using for inference. I used A100 for training.
> > For `pycocotools`, I think you don't need to install the specific version, please just use `pip install pycocotools` to install it.
> 
> ![image](https://private-user-images.githubusercontent.com/138416772/385839602-e9f7efdc-71b4-49f9-89c1-61a290fa328e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE1MTQ0NjksIm5iZiI6MTczMTUxNDE2OSwicGF0aCI6Ii8xMzg0MTY3NzIvMzg1ODM5NjAyLWU5ZjdlZmRjLTcxYjQtNDlmOS04OWMxLTYxYTI5MGZhMzI4ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExM1QxNjA5MjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jY2VlOTE1MGNhNDQ1N2JkZmNlMWViNDk0ZGZkMDY2MWI4Njk0MzAzOGNmMDM4ZjBiMmY5NjkxOTJmNjQ0MWQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HKKek-vvIBrlMlqgg0MertybVi2vlejFrbLq_srHxRo) "I noticed your YAML file specifies versions, and after a day of trying, I finally succeeded. Is this outcome expected? I have reduced the dataset."

"I feel that many coral reefs haven't been labeled."

**Comment by zhengziqiang** [comment]
What hyper-parameters are you using? Please adjust the `--iou_threshold 0.72 --sta_threshold 0.62` to lower values to get more coral reef masks, for example `--iou_threshold 0.62 --sta_threshold 0.62` or  `--iou_threshold 0.62 --sta_threshold 0.52`. It will yield more coral reef masks even under the occlusion.

**Comment by zhengziqiang** [comment]
The example images are from coral reef biologists, these coral reef images were not included in our CoralMask dataset.

**Comment by JarvisAI-7** [comment]
> these coral reef images were not included in our CoralMask dataset

"Dear friend, thank you for your prompt response. However, I'm a bit unclear about what you mean by 'these coral reef images were not included in our CoralMask dataset.' The images I shared are all from the dataset you provided. Also, I have a small request—would it be possible to make the training code public? I would be very grateful if that’s possible."
